code/controllers/subsystem/ticker.dm

```diff
@@ -71,14 +71,19 @@ var/datum/subsystem/ticker/ticker
 		syndicate_code_response	= generate_code_phrase()
 	..()
 	start_at = world.time + (config.lobby_countdown * 10)
-	world << "<span class='boldnotice'>Welcome to [station_name()]!</span>"
 	world << "Please set up your character and select \"Ready\". The game will start in about [config.lobby_countdown] seconds."
-	current_state = GAME_STATE_PREGAME
 	for(var/client/C in clients)
 		window_flash(C, ignorepref = TRUE) //let them know lobby has opened up.
 
 /datum/subsystem/ticker/fire()
 	switch(current_state)
+		if(GAME_STATE_STARTUP)
+			if(Master.initializations_finished_with_no_players_logged_in)
+				world << "Please set up your character and select \"Ready\". The game will start in about [config.lobby_countdown] seconds."
```

Comment:
I think this should always show here after initialization are done, instead of using the config, use the remaining time left.